strategy.json

```diff
@@ -1,6 +1,12 @@
 {
   "matrix": {
     "os": ["macos-latest", "ubuntu-latest", "windows-latest"],
-    "python": ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
+    "python": ["3.7", "3.8", "3.9", "3.10", "3.11"],
+    "include": [
+      {
+        "os": "ubuntu-20.04",
+	"python": "3.6"
```

Comment:
The spacing here looks a little weird for me. Is this a web editor or hard tabs vs spaces inconsistency?

Reply:
Totally just `vim` throwing a tab here where I didn't want one. Nice catch.